src/libs/services/NetService.ts

```diff
@@ -27,7 +27,17 @@ const configureService = (client?: CozyClient): void => {
 
 export const useNetService = (client?: CozyClient): void =>
   useEffect(() => {
-    configureService(client)
+    const configure = (): void => {
+      configureService(client)
+    }
+
+    client?.on('logout', configure)
```

Comment:
A good solution I didn't know we had events like that 👍.
Are we sure that the client ref in the hook won't change on the logout event? I think it should not, but if it did that would configure twice.